ext/openssl/tests/openssl_csr_export_to_file_basic.phpt

```diff
@@ -0,0 +1,76 @@
+--TEST--
+openssl_csr_export_to_file() tests
+--SKIPIF--
+<?php if (!extension_loaded("openssl")) print "skip"; ?>
+--FILE--
+<?php
+$csrfile = dirname(__FILE__) . "/openssl_csr_export_to_file_csr.tmp";
+$wrong = "wrong";
+$config = __DIR__ . DIRECTORY_SEPARATOR . 'openssl.cnf';
+$config_arg = array('config' => $config);
+
+$dn = array(
+	"countryName" => "BR",
+	"stateOrProvinceName" => "Rio Grande do Sul",
+	"localityName" => "Porto Alegre",
+	"commonName" => "Henrique do N. Angelo",
+	"emailAddress" => "hnangelo@php.net"
+);
+
+$args = array(
+	"digest_alg" => "sha1",
+	"private_key_bits" => 2048,
+	"private_key_type" => OPENSSL_KEYTYPE_DSA,
+	"encrypt_key" => true,
+	"config" => $config,
+);
+
+$privkey = 'file://' . dirname(__FILE__) . '/private_rsa_2048.key';
+$csr = openssl_csr_new($dn, $privkey, $args);
+$privkey = openssl_pkey_new($config_arg);
```

Comment:
Could we not generate generate a new key in here. The problem is that it takes much longer the supplying all params and we have got already problems with travis time outs. We really need to reduce time or openssl tests.